src/Common/tests/System/Net/WebSockets/WebSocketCreateTest.cs

```diff
@@ -151,6 +151,12 @@ public async Task ReceiveAsync_ServerSplitHeader_ValidDataReceived()
         [MemberData(nameof(EchoServersAndBoolean))]
         public async Task WebSocketProtocol_CreateFromConnectedStream_CloseAsyncClosesStream(Uri echoUri, bool explicitCloseAsync)
         {
+            if (PlatformDetection.IsWindows7)
+            {
+                // https://github.com/dotnet/corefx/issues/42339
+                return;
+            }
```

Comment:
Can we make this a conditional theory instead? e.g.
```C#
[ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindows7)]
```

Reply:
Yes it could be done with a ConditionalTheory. But we only do that when we permanently skip a test because it should never run when certain conditions are met.

Instead, this test is basically disabled like an "ActiveIssue" against an issue number. I hope to investigate the root cause and re-enable the test. If it turns out that Windows 7 has some fundamental issue that can't be solved, then I will change it to a ConditionalTheory as you suggest.